bininfo/bininfo.go

```diff
@@ -25,8 +25,8 @@
 // around our internal libraries (which use this data, e.g. to construct
 // User-Agent headers or log messages).
 //
-// When using <https://github.com/sapcc/go-makefile-maker>, go-makefile-maker
-// will detect when go-api-declarations is listed as a dependency in the
+// When using [go-makefile-maker](https://github.com/sapcc/go-makefile-maker),
+// it will detect when go-api-declarations is listed as a dependency in the
```

Comment:
This was intentionally written this way. Godoc does not support Markdown, so whatever you write here will be displayed pretty much verbatim.